frontend/plan/components/Cart.tsx

```diff
@@ -86,24 +99,12 @@ const Cart = ({
     </section>
 );
 
-Cart.propTypes = {
-    courses: PropTypes.arrayOf(PropTypes.object).isRequired,
-    toggleCourse: PropTypes.func.isRequired,
-    removeItem: PropTypes.func.isRequired,
-    courseInfo: PropTypes.func.isRequired,
-    courseInfoLoading: PropTypes.bool,
-    setTab: PropTypes.func,
-    lastAdded: PropTypes.objectOf(PropTypes.string),
-    mobileView: PropTypes.bool,
-};
-
-// const mapStateToProps = ({ schedule: { cartSections, schedules, scheduleSelected } }) => ({
 const mapStateToProps = ({
     schedule: { cartSections = [], schedules, scheduleSelected, lastAdded },
     sections: { courseInfoLoading },
-}) => ({
+}: any) => ({
     courseInfoLoading,
-    courses: cartSections.map((course) => ({
+    courses: cartSections.map((course: any) => ({
```

Comment:
I know I said not to add any types in the redux things, but i think it would be useful to add some types on these new variables. For example, we know `course` is a `Section` (and would probably be good to change its name in the process)